Makefile

```diff
@@ -2,6 +2,8 @@ REPO_DIRS=./acm-repos
 AUTO_DEPLOY_CONTEXT=kf-ci-v1
 KFCI_CONTEXT=kf-ci-v1
 
+OPTIONAL_TEST_INFRA_REPO_DIRS=./aws/GitOps
```

Comment:
can you move changes here to `aws/Makefile`?
I think the best would be fully separate the optional test infra,
e.g. you might grant additional OWNERS to `aws` folder so others can help you administer the configs

Reply:
Yeah that sounds reasonable